types/benches/batch_digest.rs

```diff
@@ -1,44 +0,0 @@
-// Copyright (c) 2022, Mysten Labs, Inc.
-// SPDX-License-Identifier: Apache-2.0
-use criterion::{
-    criterion_group, criterion_main, BenchmarkId, Criterion, SamplingMode, Throughput,
-};
-use fastcrypto::Hash;
-use rand::Rng;
-use types::{serialized_batch_digest, Batch, WorkerMessage};
-
-pub fn batch_digest(c: &mut Criterion) {
```

Comment:
This is the benchmark that checks the equivalence in performance between computing on a serialized batch and a deserialized one. This piece of code gets us away from a Great Debate over whether we should define the digest over the deserialized form of the Batch (as it is with nearly every other piece of data in the code base) or if we should only define it over the serialized bytes (to save on having to deserialize for the sole purpose of computing a digest).

TL;DR: it's important to not lose this code.